x-pack/plugins/stack_connectors/server/connector_types/crowdstrike/crowdstrike.ts

```diff
@@ -199,13 +200,26 @@ export class CrowdstrikeConnector extends SubActionConnector<
   }
 
   protected getResponseErrorMessage(
-    error: AxiosError<{ errors: [{ message: string; code: number }] }>
+    error: AxiosError<{ errors: Array<{ message: string; code: number }> }>
   ): string {
     const errorData = error.response?.data?.errors?.[0];
     if (errorData) {
       return errorData.message;
     }
 
+    // @ts-expect-error TODO: Can't find a way to make the types work here
+    const cause = isArray(error.cause?.errors) ? error.cause.errors[0] : error.cause;
+    if (cause) {
+      // ENOTFOUND is the error code for when the host is unreachable eg. api.crowdstrike.com111
+      if (cause.code === 'ENOTFOUND') {
+        return `URL not found: ${cause.hostname}`;
```

Comment:
I cannot find this property in Node.js [documentation](https://nodejs.org/docs/latest-v20.x/api/errors.html#class-systemerror). Does it exist?